parts/header-large-dark.html

```diff
@@ -1,7 +1,9 @@
 <!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|background"}}},"spacing":{"padding":{"top":"0px","bottom":"0px"},"margin":{"bottom":"var(--wp--custom--spacing--large, 8rem)"}}},"backgroundColor":"foreground","textColor":"background","layout":{"inherit":true}} -->
-<div class="wp-block-group alignfull has-background-color has-foreground-background-color has-text-color has-background has-link-color" style="margin-bottom:var(--wp--custom--spacing--large, 8rem);padding-top:0px;padding-bottom:0px"><!-- wp:template-part {"slug":"header","tagName":"header","align":"wide"} /-->
+<div class="wp-block-group"><!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|background"}}},"spacing":{"padding":{"top":"0px","bottom":"var(--wp--custom--spacing--large, 8rem)"}}},"backgroundColor":"foreground","textColor":"background","layout":{"inherit":true}} -->
+<div class="wp-block-group alignfull has-background-color has-foreground-background-color has-text-color has-background has-link-color" style="padding-top:0px;padding-bottom:var(--wp--custom--spacing--large, 8rem)"><!-- wp:template-part {"slug":"header","tagName":"header","align":"wide"} /-->
```

Comment:
You'll notice that I'm adding some bottom padding here. The bird image is supposed to have some padding underneath it, but it was removed erroneously at some point. 🤷‍♂️

The padding [is still present in the pattern](https://github.com/WordPress/twentytwentytwo/blob/trunk/inc/patterns/header-large-dark.php#L12-L13), as it should be. So I'm just adding it back in to the template too. 

Before|After
---|---
<img width="374" alt="Screen Shot 2022-01-10 at 11 33 20 AM" src="https://user-images.githubusercontent.com/1202812/148803558-b75f2ef2-7eaa-4051-a6f7-c99a88ae883e.png">|<img width="374" alt="Screen Shot 2022-01-10 at 11 29 13 AM" src="https://user-images.githubusercontent.com/1202812/148803645-dea1dabe-2e71-41a5-8b5a-887c6d8ea7b8.png">